### PR TITLE
Allows filter by artist

### LINKED ISSUE #83: Search artist name too
Currently the search box only searches the mix's title, it would be nice if it searched the artist's name too.

### PR DESCRIPTION
**Pull request by gokulchandra**
Fixes #83  , Enabling users to filter tracks by artist name.

### CODE REVIEW

**Comment by rwieruch** [comment]
That's awesome! @MiniPekka do you have time to check this PR? :)

**Comment by chriszhangusc** [comment]
Yeah I'm on it. Sorry I was a little bit busy recently.

File: src/components/StreamActivities/index.js [hunk 1]
```diff
@@ -5,13 +5,14 @@ import { bindActionCreators } from 'redux';
...
 import * as paginateLinkTypes from '../../constants/paginateLinkTypes';
-import { getAndCombined } from '../../services/filter';
+import { getAndCombined, getOrCombined } from '../../services/filter';
 import Activities from '../../components/Activities';
 import LoadingSpinner from '../../components/LoadingSpinner';
 import StreamInteractions from '../../components/StreamInteractions';
 import { DURATION_FILTER_FUNCTIONS } from '../../constants/durationFilter';
 import { getTracknameFilter } from '../../constants/nameFilter';
 import { SORT_FUNCTIONS } from '../../constants/sort';
+import { getArtistFilter } from "../../constants/artistFilter";
```
Comment (chriszhangusc) [inline src/components/StreamActivities/index.js]: '' instead of "" for consistency.

**Review by chriszhangusc — CHANGES_REQUESTED** [review]
Looks good and works. Just change the "" to ''.

File: src/components/StreamActivities/index.js (hunk 1 above)
Comment (gokulchandra) [reply src/components/StreamActivities/index.js]: Done in latest commit.

**Review by chriszhangusc — APPROVED** [review]

**Pull request merged**